README.md

```diff
@@ -34,7 +35,7 @@
 
 # 📖 Portfolio: finish mobile version <a name="about-project"></a>
```

Comment:
_**I recommend that you change the title "Portfolio: finish mobile version" of this session, since your new project is not called that way. I could name it as: Review one of your past projects or Review of presonal project.**_